.github/workflows/validate_markdown_links.json

```diff
@@ -2,7 +2,7 @@
     "ignorePatterns": [
         {
             "description": "Ignore anchor links that currently cannot be resolved. See https://github.com/tcort/markdown-link-check/issues/225",
-            "pattern": "^#"
+            "pattern": "^(?!http).+#.+"
```

Comment:
we also need to ignore anchor links to the same document, like "#abc" (which is the only case that's supported by the current regex `"^#"`)

Reply:
Fixed